src/components/BounceRulesActivity/index.scss

```diff
@@ -0,0 +1,69 @@
+.container {
```

Comment:
Let's rename some of these classes to be more BEM-like. container is a bit too generic and has a possibility to spread and affect other containers on the page and we can add a class to h1 as well to not be as specific as the h1 element specificity is the highest and will override too many things